src/style.css

```diff
@@ -214,3 +225,62 @@ button {
   color: transparent;
   text-align: center;
 }
+
+.cat-popup {
+  display: flex;
+  flex-wrap: wrap;
+  position: fixed;
+  top: 10px;
+  left: 10px;
+  background-color: white;
+  border: 4px solid #3e3e3e;
+  overflow-y: scroll;
+  height: 100%;
+  z-index: 999;
+  padding: 0 32px 0 32px;
+}
+
```

Comment:
good job 👍, that you created a category popup it's not mandatory or from the requirements of the project, I suggest adding more style for it to make it look great and more organized. 😃 [optional]

Reply:
sure. thanks!